### DISCUSSION TITLE
v5 dagre图 布局问题

### DISCUSSION THREAD

**Discussion by rendeyuwei** [antvis/G6#5766] [Q&A] [answered]
看到dagre的layout配置下的align属性支持这几个值：
![image](https://github.com/antvis/G6/assets/52694265/6431d793-0702-424b-ab13-3866cbba1d35)
想问下是否可以支持类似正上方对齐，比如就叫做U（表示正上方对齐），目前四个值只有四个角对齐

**Comment by Aarebecca** [comment, marked as answer]
目前 v5 dagre 使用了 dagre.js 布局，相关问题请移至：https://github.com/dagrejs/dagre/issues
